src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -37,12 +40,12 @@
 
 public class QuoraProfileScraper extends BaseScraper {
 
-    private final long serialVersionUID = -3398701925784347310L;
+    private final long serialVersionUID = -3398701925784347312L;
```

Comment:
I have done this to show that this class is not backward compatible. 
https://stackoverflow.com/questions/285793/what-is-a-serialversionuid-and-why-should-i-use-it